lib/rules/linebreaks/index.js

```diff
@@ -32,13 +32,13 @@ function rule(actual, secondary, context) {
 
 			root.walk((node) => {
 				propertiesToUpdate.forEach((property) => {
-					const fixedData = fixData(_.get(node, property), shouldHaveCR);
+					const fixedData = fixData(_.get(node, property));
```

Comment:
[note] `shouldHaveCR` defined in the outer scope is available.

See also:

https://github.com/stylelint/stylelint/blob/caccf9a660833fdaeaae96721ca087122825e2b0/lib/rules/linebreaks/index.js#L28